### DISCUSSION TITLE
How to search GO term obtained in 2020

### DISCUSSION THREAD

**Discussion by wcc-new** [geneontology/helpdesk#405] [Q&A]
I performed annotation using Trinotate end of 2020 which used UniProt Knowledgebase (UniProtKB) Release 2020_05. 

I would like to retrieve list of GO term related to specific key word using [geneontology](http://geneontology.org/). However, it is using the latest release (Current release 2022-07-01)which I afraid there might be some changes in the GO term as different version of release is used. May I know is there anyway to retrieve GO term list based on **release of 2020** by typing the key word as shown in the figure below?

![Picture2](https://user-images.githubusercontent.com/55751222/181149196-435951a0-09d2-4c2f-81ba-473813c921bf.jpg)

**Comment by balhoff** [comment]
@wcc-new here is the GO release archive: http://release.geneontology.org

You can find releases from various dates there.

**Reply by wcc-new** [reply]
Hi 

I did aware I can retrieve those GO from the archive. However, is there anyway to retrieve the GO term list by keyword search as shown in the figure as posted earlier from the Gene ontology archived in 2020?

**Comment by kltm** [comment]
@wcc-new 
To answer your original question: not directly. However, as mentioned by @balhoff , we offer our data products going back some time and out could setup the AmiGO software stack by grabbing the appropriate solr/golr index and running it. Something like:

```
mkdir -p /tmp/srv-solr-data/index
cd /tmp/srv-solr-data/index
wget http://release.geneontology.org/YOUR_DATE_HERE/products/solr/golr-index-contents.tgz
tar -zxvf golr-index-contents.tgz
docker run -p 8080:8080 -p 9999:9999 -v /tmp/srv-solr-data:/srv/solr/data -t geneontology/amigo-standalone
```

**Reply by wcc-new** [reply]
thanks for the guidance. I will try this command line method.
